README.md

```diff
@@ -77,53 +68,73 @@ build/$(go env GOOS)/gotagger
 v0.4.0
 ```
 
-**Note**: The version reported may be different,
-depending on what unreleased changes exist.
+**Note**: The version reported may be different, depending on what unreleased
+changes exist.
 
-To tag a release,
-make any changes needed to prepare your project for releasing
-(ie. update the change log,
-merge any feature branches).
-Then create a "release" commit and run gotagger again:
+To tag a release, make any changes needed to prepare your project for releasing
+(ie. update the change log, merge any feature branches). Then create a "release"
+commit and run gotagger again:
 
 ```bash
 VERSION="$(gotagger)"
 git commit -m "release: $VERSION"
 gotagger -release
 ```
 
-You can now perform any release builds,
-push the tag to your central git repository,
-or any other post-release tasks.
+You can now perform any release builds, push the tag to your central git
+repository, or any other post-release tasks.
 
-`gotagger` can also push any tags it creates,
-by using the `-push` flag.
+`gotagger` can also push any tags it creates, by using the `-push` flag.
 
 ```bash
 gotagger -release -push
 ```
 
+#### Customizing Version Increments
+
+In some cases, you may wish to change which semantic version field should
+increment based on which commit type is provided. You can do this via
+a [config file](gotagger.json). The config file contains a mapping of commit
+type to semver increment and a default increment to use if it encounters an
+unknown commit type. For example:
+
+```json
+{
+  "incrementMappings": {
+    "release": "major",
```

Comment:
"release" means something specific to gotagger, so I don't think we should allow it in the mapping. "release" must at least be "patch", or else gotagger will try to create the same tag twice (and fail) if you try to do two release commits back-to-back. If "release" is "minor" or "major", then you will never be able to release any less significant version increment.

Also the conventional commit spec is explicit about how to increment the major version, so should gotagger even let you configure a "major" increment?

Reply:
I changed it so release and major increments can't be defined in the increment config.